frontends/mit-learn/webpack.config.js

```diff
@@ -182,7 +182,16 @@ module.exports = (env, argv) => {
         },
         {
           test: /\.tsx?$/,
-          use: "swc-loader",
+          use: {
+            loader: "swc-loader",
+            options: {
+              jsc: {
+                experimental: {
+                  plugins: [["@swc/plugin-emotion", {}]],
+                },
+              },
+            },
+          },
```

Comment:
This is https://www.npmjs.com/package/@swc/plugin-emotion

In particular, in dev mode, this gives us more useful class names. (production is unchanged).